app/pages/home-for-user/index.jsx

```diff
@@ -120,10 +125,36 @@ const HomePageForUser = React.createClass({
     });
   },
 
+  recentSubjects(ribbonData) {
+    this.setState({
+      newData: this.fetchRecentSubjects(ribbonData),
+    });
+  },
+
   findProjectLink(project) {
     return `/projects/${project.slug}`;
   },
 
+  fetchRecentSubjects(ribbonData) {
+    const newestSubjects = []
+    ribbonData.map((project) => {
+      if (project['recentSubjectSet'] !== undefined){
+        newestSubjects.push(project)
+      }
+    });
+    const recentSets = newestSubjects.sort((a, b) => {
```

Comment:
Also, the naming here is a bit confusing... we're sorting subject sets, not subjects, correct?